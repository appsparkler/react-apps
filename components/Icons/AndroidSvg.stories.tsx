import type { Meta, StoryObj } from "@storybook/react";
import { AndroidSvg } from "./AndroidSvg";
// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
const meta: Meta<typeof AndroidSvg> = {
  title: "Components/SVG Files/Android",
  component: AndroidSvg,
  tags: ["autodocs"],
  argTypes: {},
};
export default meta;
type Story = StoryObj<typeof AndroidSvg>;
// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
export const AndroidSvgStory: Story = {
  args: {
    sx: {
      width: 300,
      height: 70,
    },
  },
};

AndroidSvgStory.storyName = "Android";
