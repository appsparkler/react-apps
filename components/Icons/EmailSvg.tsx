"use client";
import { SvgIcon, SvgIconProps } from "@mui/material";

export const EmailSvgIcon: React.FC<SvgIconProps> = (props) => {
  return (
    <SvgIcon
      sx={{ height: 28, width: 28 }}
      viewBox="0 0 34 34"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
    >
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M4.25 4.25H29.75C31.3148 4.25 32.5833 5.51853 32.5833 7.08333V26.9167C32.5833 28.4815 31.3148 29.75 29.75 29.75H4.25C2.68519 29.75 1.41666 28.4815 1.41666 26.9167V7.08333C1.41666 5.51853 2.68519 4.25 4.25 4.25ZM4.25 13.6257V26.9167H29.75V13.6263L17 20.0013L4.25 13.6257ZM4.25 10.4578L17 16.8335L29.75 10.4585V7.08333H4.25V10.4578Z"
        fill="url(#paint0_linear_14_52)"
      />
      <defs>
        <linearGradient
          id="paint0_linear_14_52"
          x1="32.5833"
          y1="16.939"
          x2="1.41666"
          y2="16.939"
          gradientUnits="userSpaceOnUse"
        >
          <stop stopColor="#2EF20E" />
          <stop offset="0.584338" stopColor="#0EF2E5" />
        </linearGradient>
      </defs>
    </SvgIcon>
  );
};
