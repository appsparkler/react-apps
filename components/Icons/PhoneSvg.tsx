import { SvgIcon, SvgIconProps } from "@mui/material";

export const PhoneSvgIcon: React.FC<SvgIconProps> = (props) => {
  return (
    <SvgIcon
      sx={{ height: 28, width: 28 }}
      viewBox="0 0 34 34"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
    >
      <path
        fill-rule="evenodd"
        clip-rule="evenodd"
        d="M15.3802 11.8121C16.1141 10.823 16.3773 9.78055 15.5472 8.86038C13.6844 6.25375 12.4316 4.64181 11.6457 3.86554C10.1518 2.38975 7.69417 2.58937 6.4003 3.86428C5.70534 4.54904 5.47007 4.78419 4.7565 5.50897C0.78183 9.48578 3.20579 17.8926 9.64956 24.3427C16.0919 30.7914 24.4979 33.2166 28.4809 29.2314C29.1394 28.5966 29.6969 28.0388 30.1366 27.5741C31.4045 26.2339 31.597 23.8847 30.1282 22.3585C29.3752 21.5761 27.8381 20.3838 25.1218 18.4408C24.2898 17.6969 23.3034 17.8581 22.3991 18.4514C21.9641 18.7368 21.6475 19.0255 21.0497 19.6239L19.9642 20.71C19.8212 20.853 17.8798 19.8808 15.9953 17.9944C14.1097 16.1069 13.1381 14.1654 13.2804 14.0232C13.5739 13.7297 13.936 13.3674 14.3667 12.9363C14.5563 12.7465 14.647 12.6545 14.7633 12.5323C15.0056 12.2777 15.2061 12.0466 15.3802 11.8121ZM21.9666 22.7134L23.0523 21.6271C23.3813 21.2978 23.5945 21.0953 23.7679 20.9557C26.1474 22.6645 27.5255 23.739 28.0884 24.3238C28.4264 24.675 28.3739 25.3157 28.0801 25.6263C27.6735 26.056 27.1453 26.5845 26.4971 27.2097C23.9215 29.7864 17.136 27.8287 11.6524 22.3397C6.16705 16.849 4.21026 10.0624 6.76637 7.50488C7.47706 6.78308 7.70223 6.55803 8.38726 5.88304C8.64411 5.62995 9.34375 5.57313 9.6562 5.88178C10.2612 6.47943 11.3835 7.91643 13.0344 10.2162C12.949 10.3223 12.8422 10.4422 12.7123 10.5787C12.617 10.6789 12.5375 10.7594 12.3639 10.9333C11.9335 11.3641 11.5717 11.726 11.2787 12.019C9.43256 13.865 11.0048 17.0068 13.9925 19.9974C16.9781 22.986 20.121 24.56 21.9666 22.7134Z"
        fill="url(#paint0_linear_14_51)"
      />
      <defs>
        <linearGradient
          id="paint0_linear_14_51"
          x1="31.1598"
          y1="16.9324"
          x2="2.82651"
          y2="16.9324"
          gradientUnits="userSpaceOnUse"
        >
          <stop stop-color="#2EF20E" />
          <stop offset="0.584338" stop-color="#0EF2E5" />
        </linearGradient>
      </defs>
    </SvgIcon>
  );
};
