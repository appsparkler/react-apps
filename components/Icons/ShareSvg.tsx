import { SvgIcon, SvgIconProps } from "@mui/material";

export const ShareSvgIcon: React.FC<SvgIconProps> = (props) => {
  return (
    <SvgIcon
      sx={{ height: 28, width: 28 }}
      viewBox="0 0 34 34"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
    >
      <path
        fill-rule="evenodd"
        clip-rule="evenodd"
        d="M19.8333 8.50001C19.8333 8.90719 19.8763 9.30434 19.9579 9.68718L12.774 13.2791C11.7351 12.0868 10.2056 11.3333 8.5 11.3333C5.37039 11.3333 2.83334 13.8704 2.83334 17C2.83334 20.1296 5.37039 22.6667 8.5 22.6667C10.2056 22.6667 11.7351 21.9132 12.774 20.7209L19.9579 24.3128C19.8763 24.6957 19.8333 25.0928 19.8333 25.5C19.8333 28.6296 22.3704 31.1667 25.5 31.1667C28.6296 31.1667 31.1667 28.6296 31.1667 25.5C31.1667 22.3704 28.6296 19.8333 25.5 19.8333C23.7944 19.8333 22.2649 20.5868 21.226 21.7791L14.0421 18.1872C14.1237 17.8043 14.1667 17.4072 14.1667 17C14.1667 16.5928 14.1237 16.1957 14.0421 15.8128L21.226 12.2209C22.2649 13.4132 23.7944 14.1667 25.5 14.1667C28.6296 14.1667 31.1667 11.6296 31.1667 8.50001C31.1667 5.3704 28.6296 2.83334 25.5 2.83334C22.3704 2.83334 19.8333 5.3704 19.8333 8.50001ZM11.3333 17C11.3333 18.5648 10.0648 19.8333 8.5 19.8333C6.9352 19.8333 5.66667 18.5648 5.66667 17C5.66667 15.4352 6.9352 14.1667 8.5 14.1667C10.0648 14.1667 11.3333 15.4352 11.3333 17ZM25.5 11.3333C27.0648 11.3333 28.3333 10.0648 28.3333 8.50001C28.3333 6.9352 27.0648 5.66668 25.5 5.66668C23.9352 5.66668 22.6667 6.9352 22.6667 8.50001C22.6667 10.0648 23.9352 11.3333 25.5 11.3333ZM28.3333 25.5C28.3333 27.0648 27.0648 28.3333 25.5 28.3333C23.9352 28.3333 22.6667 27.0648 22.6667 25.5C22.6667 23.9352 23.9352 22.6667 25.5 22.6667C27.0648 22.6667 28.3333 23.9352 28.3333 25.5Z"
        fill="url(#paint0_linear_14_53)"
      />
      <defs>
        <linearGradient
          id="paint0_linear_14_53"
          x1="31.1667"
          y1="16.9322"
          x2="2.83334"
          y2="16.9322"
          gradientUnits="userSpaceOnUse"
        >
          <stop stop-color="#2EF20E" />
          <stop offset="0.584338" stop-color="#0EF2E5" />
        </linearGradient>
      </defs>
    </SvgIcon>
  );
};
