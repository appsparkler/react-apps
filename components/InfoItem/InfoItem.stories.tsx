import type { Meta, StoryObj } from "@storybook/react";
import { InfoItem, InfoSection } from "./InfoItem";
import { stringResources } from "../../values/strings";
import { PhoneSvgIcon } from "../Icons/PhoneSvg";
import { EmailSvgIcon } from "../Icons/EmailSvg";
import { ShareSvgIcon } from "../Icons/ShareSvg";
import { Box } from "@mui/material";
// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
const meta: Meta<typeof InfoItem> = {
  title: "Components/Info Item",
  component: InfoItem,
  tags: ["autodocs"],
  argTypes: {},
};

export default meta;
type Story = StoryObj<typeof InfoItem>;

// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args

export const Mobile: Story = {
  args: {
    Icon: PhoneSvgIcon,
    info: stringResources.mobile,
  },
};

export const Email: Story = {
  args: {
    Icon: EmailSvgIcon,
    info: stringResources.email,
  },
};

export const Social: Story = {
  args: {
    Icon: ShareSvgIcon,
    info: stringResources.social,
  },
};

export const infoSection = () => <InfoSection />;
