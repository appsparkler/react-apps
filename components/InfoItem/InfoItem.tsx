import { Box, Typography } from "@mui/material";
import React from "react";
import { InfoItemFC } from "../../types";
import { PhoneSvgIcon } from "../Icons/PhoneSvg";
import { EmailSvgIcon } from "../Icons/EmailSvg";
import { ShareSvgIcon } from "../Icons/ShareSvg";
import { stringResources } from "../../values/strings";

export const InfoItem: InfoItemFC = ({ Icon, info }) => {
  return (
    <>
      <Box
        minHeight="2px"
        sx={{
          background: (theme) =>
            `linear-gradient(90deg, ${theme.palette.secondary.main} 0%, ${theme.palette.primary.main} 100%)`,
        }}
      />
      <Box
        py={2}
        pl="20%"
        display="flex"
        flexDirection={"row"}
        alignItems={"center"}
      >
        <Icon sx={{ height: 28, width: 28 }} />
        <Typography variant="h6" ml={1}>
          {info}
        </Typography>
      </Box>
    </>
  );
};

export const InfoSection = () => (
  <Box display={"flex"} flexDirection={"column"}>
    <InfoItem Icon={PhoneSvgIcon} info={stringResources.mobile} />
    <InfoItem Icon={EmailSvgIcon} info={stringResources.email} />
    <InfoItem Icon={ShareSvgIcon} info={stringResources.social} />
  </Box>
);
