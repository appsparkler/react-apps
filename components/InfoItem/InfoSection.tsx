import { Box, BoxProps } from "@mui/material";
import { InfoItem } from "./InfoItem";
import { PhoneSvgIcon } from "../Icons/PhoneSvg";
import { stringResources } from "@/values/strings";
import { ShareSvgIcon } from "../Icons/ShareSvg";
import { EmailSvgIcon } from "../Icons/EmailSvg";

export const InfoSection: React.FC<{ wrapperBoxProps?: BoxProps }> = ({
  wrapperBoxProps,
}) => (
  <Box display={"flex"} flexDirection={"column"} {...wrapperBoxProps}>
    <InfoItem Icon={PhoneSvgIcon} info={stringResources.mobile} />
    <InfoItem Icon={EmailSvgIcon} info={stringResources.email} />
    <InfoItem Icon={ShareSvgIcon} info={stringResources.social} />
  </Box>
);
