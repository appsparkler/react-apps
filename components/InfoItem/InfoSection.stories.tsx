import type { Meta, StoryObj } from "@storybook/react";
import { InfoSection } from "./InfoSection";
// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
const meta: Meta<typeof InfoSection> = {
  title: "Components/Info Section",
  component: InfoSection,
  tags: ["autodocs"],
  argTypes: {},
};
export default meta;
type Story = StoryObj<typeof InfoSection>;
// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
export const InfoSectionStory: Story = {
  args: {},
};
InfoSectionStory.storyName = "Info Section";
