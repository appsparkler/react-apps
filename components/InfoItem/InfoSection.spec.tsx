import { Box } from "@mui/material";
import { cleanup, render } from "@testing-library/react";
import { InfoItem } from "./InfoItem";
import { PhoneSvgIcon } from "../Icons/PhoneSvg";
import { stringResources } from "@/values/strings";
import { ShareSvgIcon } from "../Icons/ShareSvg";
import { EmailSvgIcon } from "../Icons/EmailSvg";
import { screen } from "@testing-library/react";

afterEach(cleanup);

const InfoSection = () => (
  <Box display={"flex"} flexDirection={"column"}>
    <InfoItem Icon={PhoneSvgIcon} info={stringResources.mobile} />
    <InfoItem Icon={EmailSvgIcon} info={stringResources.email} />
    <InfoItem Icon={ShareSvgIcon} info={stringResources.social} />
  </Box>
);

describe("InfoSection Snapshots", () => {
  it("should match snapshot", () => {
    const { container } = render(<InfoSection />);
    expect(container).toMatchSnapshot();
  });
});

describe("InfoSection functional tests", () => {
  it("should render name, email and mobile number", () => {
    render(<InfoSection />);
    expect(screen.getByText(stringResources.email)).toBeVisible();
    expect(screen.getByText(stringResources.mobile)).toBeVisible();
    expect(screen.getByText(stringResources.social)).toBeVisible();
  });
});
