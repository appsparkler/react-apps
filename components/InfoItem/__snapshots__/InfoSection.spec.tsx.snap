// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoSection Snapshots should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-j7qwjs"
  >
    <div
      class="MuiBox-root css-1ong228"
    />
    <div
      class="MuiBox-root css-14f46cv"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oazvwu-MuiSvgIcon-root"
        fill="none"
        focusable="false"
        viewBox="0 0 34 34"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M15.3802 11.8121C16.1141 10.823 16.3773 9.78055 15.5472 8.86038C13.6844 6.25375 12.4316 4.64181 11.6457 3.86554C10.1518 2.38975 7.69417 2.58937 6.4003 3.86428C5.70534 4.54904 5.47007 4.78419 4.7565 5.50897C0.78183 9.48578 3.20579 17.8926 9.64956 24.3427C16.0919 30.7914 24.4979 33.2166 28.4809 29.2314C29.1394 28.5966 29.6969 28.0388 30.1366 27.5741C31.4045 26.2339 31.597 23.8847 30.1282 22.3585C29.3752 21.5761 27.8381 20.3838 25.1218 18.4408C24.2898 17.6969 23.3034 17.8581 22.3991 18.4514C21.9641 18.7368 21.6475 19.0255 21.0497 19.6239L19.9642 20.71C19.8212 20.853 17.8798 19.8808 15.9953 17.9944C14.1097 16.1069 13.1381 14.1654 13.2804 14.0232C13.5739 13.7297 13.936 13.3674 14.3667 12.9363C14.5563 12.7465 14.647 12.6545 14.7633 12.5323C15.0056 12.2777 15.2061 12.0466 15.3802 11.8121ZM21.9666 22.7134L23.0523 21.6271C23.3813 21.2978 23.5945 21.0953 23.7679 20.9557C26.1474 22.6645 27.5255 23.739 28.0884 24.3238C28.4264 24.675 28.3739 25.3157 28.0801 25.6263C27.6735 26.056 27.1453 26.5845 26.4971 27.2097C23.9215 29.7864 17.136 27.8287 11.6524 22.3397C6.16705 16.849 4.21026 10.0624 6.76637 7.50488C7.47706 6.78308 7.70223 6.55803 8.38726 5.88304C8.64411 5.62995 9.34375 5.57313 9.6562 5.88178C10.2612 6.47943 11.3835 7.91643 13.0344 10.2162C12.949 10.3223 12.8422 10.4422 12.7123 10.5787C12.617 10.6789 12.5375 10.7594 12.3639 10.9333C11.9335 11.3641 11.5717 11.726 11.2787 12.019C9.43256 13.865 11.0048 17.0068 13.9925 19.9974C16.9781 22.986 20.121 24.56 21.9666 22.7134Z"
          fill="url(#paint0_linear_14_51)"
          fill-rule="evenodd"
        />
        <defs>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="paint0_linear_14_51"
            x1="31.1598"
            x2="2.82651"
            y1="16.9324"
            y2="16.9324"
          >
            <stop
              stop-color="#2EF20E"
            />
            <stop
              offset="0.584338"
              stop-color="#0EF2E5"
            />
          </lineargradient>
        </defs>
      </svg>
      <h6
        class="MuiTypography-root MuiTypography-h6 css-ssm9q9-MuiTypography-root"
      >
        +91-733-8080-855
      </h6>
    </div>
    <div
      class="MuiBox-root css-1ong228"
    />
    <div
      class="MuiBox-root css-14f46cv"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oazvwu-MuiSvgIcon-root"
        fill="none"
        focusable="false"
        viewBox="0 0 34 34"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M4.25 4.25H29.75C31.3148 4.25 32.5833 5.51853 32.5833 7.08333V26.9167C32.5833 28.4815 31.3148 29.75 29.75 29.75H4.25C2.68519 29.75 1.41666 28.4815 1.41666 26.9167V7.08333C1.41666 5.51853 2.68519 4.25 4.25 4.25ZM4.25 13.6257V26.9167H29.75V13.6263L17 20.0013L4.25 13.6257ZM4.25 10.4578L17 16.8335L29.75 10.4585V7.08333H4.25V10.4578Z"
          fill="url(#paint0_linear_14_52)"
          fill-rule="evenodd"
        />
        <defs>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="paint0_linear_14_52"
            x1="32.5833"
            x2="1.41666"
            y1="16.939"
            y2="16.939"
          >
            <stop
              stop-color="#2EF20E"
            />
            <stop
              offset="0.584338"
              stop-color="#0EF2E5"
            />
          </lineargradient>
        </defs>
      </svg>
      <h6
        class="MuiTypography-root MuiTypography-h6 css-ssm9q9-MuiTypography-root"
      >
        appsparkler@gmail.com
      </h6>
    </div>
    <div
      class="MuiBox-root css-1ong228"
    />
    <div
      class="MuiBox-root css-14f46cv"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oazvwu-MuiSvgIcon-root"
        fill="none"
        focusable="false"
        viewBox="0 0 34 34"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M19.8333 8.50001C19.8333 8.90719 19.8763 9.30434 19.9579 9.68718L12.774 13.2791C11.7351 12.0868 10.2056 11.3333 8.5 11.3333C5.37039 11.3333 2.83334 13.8704 2.83334 17C2.83334 20.1296 5.37039 22.6667 8.5 22.6667C10.2056 22.6667 11.7351 21.9132 12.774 20.7209L19.9579 24.3128C19.8763 24.6957 19.8333 25.0928 19.8333 25.5C19.8333 28.6296 22.3704 31.1667 25.5 31.1667C28.6296 31.1667 31.1667 28.6296 31.1667 25.5C31.1667 22.3704 28.6296 19.8333 25.5 19.8333C23.7944 19.8333 22.2649 20.5868 21.226 21.7791L14.0421 18.1872C14.1237 17.8043 14.1667 17.4072 14.1667 17C14.1667 16.5928 14.1237 16.1957 14.0421 15.8128L21.226 12.2209C22.2649 13.4132 23.7944 14.1667 25.5 14.1667C28.6296 14.1667 31.1667 11.6296 31.1667 8.50001C31.1667 5.3704 28.6296 2.83334 25.5 2.83334C22.3704 2.83334 19.8333 5.3704 19.8333 8.50001ZM11.3333 17C11.3333 18.5648 10.0648 19.8333 8.5 19.8333C6.9352 19.8333 5.66667 18.5648 5.66667 17C5.66667 15.4352 6.9352 14.1667 8.5 14.1667C10.0648 14.1667 11.3333 15.4352 11.3333 17ZM25.5 11.3333C27.0648 11.3333 28.3333 10.0648 28.3333 8.50001C28.3333 6.9352 27.0648 5.66668 25.5 5.66668C23.9352 5.66668 22.6667 6.9352 22.6667 8.50001C22.6667 10.0648 23.9352 11.3333 25.5 11.3333ZM28.3333 25.5C28.3333 27.0648 27.0648 28.3333 25.5 28.3333C23.9352 28.3333 22.6667 27.0648 22.6667 25.5C22.6667 23.9352 23.9352 22.6667 25.5 22.6667C27.0648 22.6667 28.3333 23.9352 28.3333 25.5Z"
          fill="url(#paint0_linear_14_53)"
          fill-rule="evenodd"
        />
        <defs>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="paint0_linear_14_53"
            x1="31.1667"
            x2="2.83334"
            y1="16.9322"
            y2="16.9322"
          >
            <stop
              stop-color="#2EF20E"
            />
            <stop
              offset="0.584338"
              stop-color="#0EF2E5"
            />
          </lineargradient>
        </defs>
      </svg>
      <h6
        class="MuiTypography-root MuiTypography-h6 css-ssm9q9-MuiTypography-root"
      >
        appsparkler
      </h6>
    </div>
  </div>
</div>
`;
